lib/rubocop/cop/mixin/percent_literal.rb

```diff
@@ -85,14 +86,14 @@ def autocorrect_words(node, escape, delimiters)
         end.join(' ')
       end
 
-      def line_breaks(node, source, previous_line_number, base_line_number)
+      def line_breaks(node, source, previous_line_num, base_line_num, node_idx)
```

Comment:
Why were the variables renamed from `number` to `num`?

Reply:
Adding the new `node_idx` argument to the list pushed the line length too long. Would've preferred to keep `_number` as part of the names, but I figured this was a decent compromise to get the line length back under the limit.